repository.py

```diff
@@ -791,28 +791,36 @@ def installFromYum(targets, mounts, progress_callback, cachedir):
         count = 0
         total = 0
         verify_count = 0
+        progressLine = re.compile('.*(\d+)/(\d+)$')
         while True:
             line = p.stdout.readline()
             if not line:
                 break
             line = line.rstrip()
             logger.log("DNF: %s" % line)
+            norm_line = ' '.join(line.split())
             if line == 'Resolving Dependencies':
                 progress_callback(1)
-            elif line == 'Dependencies Resolved':
+            elif line == 'Dependencies Resolved' or line == 'Dependencies resolved.':
                 progress_callback(3)
             elif line.startswith('-----------------------------------------'):
```

Comment:
The code now uses a mixture of `line` and `norm_line`. Can it just use `norm_line` everywhere?

Reply:
The checks with `line` are more strict. Potentially the code could evolve with even a more "normalized" version (for instance making all lower case) so you want to keep 2 versions.

Reply:
I don't see anywhere that `line` is currently used that doesn't exactly match the value of `norm_line` at the same point, it would be nice to use just one